assets/src/components/mapMarkers.tsx

```diff
@@ -108,7 +110,27 @@ export const VehicleMarker = ({
   onSelect?: (vehicle: Vehicle) => void
 }) => {
   const [{ userSettings }] = useContext(StateDispatchContext)
-  const eventHandlers = onSelect ? { click: () => onSelect(vehicle) } : {}
+  const markerRef = useRef<Leaflet.Marker<any>>(null)
+
+  const suppressPopup = () => {
+    const marker = markerRef.current
+    if (marker) {
+      marker.closePopup()
+    }
```

Comment:
**suggestion (non-blocking):** up to you if you feel that this is more or less readable, but it looks like TypeScript is smart enough to realize that this will work and it's somewhat more concise.

```suggestion
    markerRef.current?.closePopup()
```

Reply:
https://github.com/mbta/skate/pull/2373/commits/3c35d54b355f58d41f2797009ce6d1cdf2cfc685